circe/src/main/scala/org/http4s/circe/middleware/JsonDebugErrorHandler.scala

```diff
@@ -148,7 +148,7 @@ object JsonDebugErrorHandler {
           .map(_.toList.map(encodeThrowable(_)))
           .asJson,
         "class_name" -> Option(a.getClass())
-          .flatMap(c => Option(c.getCanonicalName()))
+          .flatMap(c => Option(c.getName()))
```

Comment:
Is canonical name not supported on scala.js?

Reply:
It's not, see https://github.com/scala-js/scala-js/issues/3452 If this is important we can split this file by platform :(

Reply:
It's, uh, usually the same.  As long as we call out the change.